runtime/vm/j9vm.tdf

```diff
@@ -1012,3 +1012,8 @@ TraceEntry=Trc_VM_snapshot_subAllocateSnapshotMemory_Entry NoEnv Overhead=1 Leve
 TraceExit=Trc_VM_snapshot_subAllocateSnapshotMemory_Exit NoEnv Overhead=1 Level=3 Template="subAllocateMemory() Memory allocated = %p."
 TraceEvent=Trc_VM_snapshot_loadWarmClassFromSnapshot_ClassLoadHookFailed Overhead=1 Level=3 Template="loadWarmClassFromSnapshot() Warm class load hook failed class=%p, %s"
 TraceEvent=Trc_VM_snapshot_loadWarmClassFromSnapshot_ClassInfo Overhead=1 Level=3 Template="loadWarmClassFromSnapshot() LoadClass clazz=%p, %s"
+
+TraceEntry=Trc_VM_getTypeIDUTF8_Entry Overhead=1 Level=5 Template="getTypeIDUTF8 className %.*s"
+TraceExit=Trc_VM_getTypeIDUTF8_Exit Overhead=1 Level=5 Template="getTypeIDUTF8 className %.*s clazz=%p result=%lli"
+TraceEntry=Trc_VM_getTypeID_Entry Overhead=1 Level=5 Template="getTypeID clazz=%p"
+TraceExit=Trc_VM_getTypeID_Exit Overhead=1 Level=5 Template="getTypeID clazz=%p result=%lli"
```

Comment:
Suggest these new tracepoint names should match the functions where they're used (`TypeID` -> `TypeId`).